src/js/utils/menu-utils.jsx

```diff
@@ -0,0 +1,80 @@
+import _ from 'lodash';
+import queryString from 'query-string';
+
+export const checkActiveSection = ({
```

Comment:
@alannadolny could this util also be simplified to use the new `sections config`?

Reply:
I don't think so, because this util has to compare sectionSearch, not only sectionPath. There are some tricky places where the sectionSearch can change the section which should be underlined.